trusted.ci.jenkins.io.tf

```
@@ -351,10 +399,10 @@ resource "azurerm_network_security_rule" "deny_all_to_vnet" {
 }
 
 # TODO: add all ips from people needing access to trusted.ci.jenkins.io, or remove the rule at final migration. (Cf https://github.com/jenkins-infra/azure/pull/334#discussion_r1179955821)
-resource "azurerm_network_security_rule" "allow_ssh_from_admins_to_bounce" {
+resource "azurerm_network_security_rule" "allow_inbound_ssh_from_admins_to_bounce" {
   for_each = local.admin_allowed_ips
 
-  name                        = "allow-22-from-${each.key}-to-bounce"
+  name                        = "allow-inbound-ssh-from-${each.key}-to-bounce"
```

```suggestion
  name                        = "allow-inbound-ssh-from-admin-${each.key}-to-bounce"
```
Suggestion, not mandatory at all